### PR TITLE
Python 3 support

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by graingert** [comment]
@srobertson bump?

**Comment by srobertson** [comment]
@graingert thanks! I need to test all these out locally and fix the unit tests. Hoping to land all your great work by the weekend.

**Comment by srobertson** [comment]
Needs a rebase to pick up unit tests fixes to python 2.7. Test were failing for me on 3.5.

**Comment by graingert** [comment]
@srobertson yup, fixed

File: shipwright/__init__.py [hunk 1]
```diff
@@ -43,10 +43,10 @@ def targets(self):
  
     commit_map = commits.mkmap(self.source_control)
 
-    last_built_ref, last_built_rel = zip(*map(
+    last_built_ref, last_built_rel = zip(*list(map(
```
Comment (srobertson) [inline shipwright/__init__.py]: Why is this necessary in python 3?

Comment (graingert) [inline shipwright/__init__.py]: technically you don't.

**Comment by srobertson** [comment]
+1, wow thank you so much for the python 3 support!!
